tasks/integer-square-root/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use num::BigUint;
+use num::CheckedSub;
+use num_traits::{One, Zero};
+
+fn isqrt(number: &BigUint) -> BigUint {
```

Comment:
I don't see much benefit in using bignums here. Can we just use `u64`/`i64`?

Reply:
According to the [task requirement](http://rosettacode.org/wiki/Isqrt_(integer_square_root)_of_X) solution should display up to square root of 7^73, which would not fit into u64.